std/node/_fs/_fs_exists_test.ts

```diff
@@ -6,10 +6,12 @@ import { exists, existsSync } from "./_fs_exists.ts";
 const { test } = Deno;
 
 test(async function existsFile() {
-  const availableFile = await new Promise(async (resolve) => {
```

Comment:
This problem should have been caught by eslint - https://eslint.org/docs/rules/no-async-promise-executor